src/assets/icons/ico-coaching.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="36" height="36" viewBox="0 0 36 36" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Looks like this icon is unused. It should probably be included in the main switch beside "Lerncoaching" as in the Figma design template.

Reply:
I removed them. They can be included at some future point. For now it is ok without the icons according to @reinerschristopher.